code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,826 @@
+					L.attack_animal(src)
+				else
+					attackstep = 0
+			else
+				L.attack_animal(src)
+		else
+			L.attack_animal(src)
+	else
+		target.attack_animal(src)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/adjustBruteLoss(var/damage)
+	..(damage)
+	Retaliate()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/adjustFireLoss(var/damage)
+	..(damage)
+	Retaliate()
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/bullet_act(var/obj/item/projectile/Proj)
+	if (istype(Proj, /obj/item/projectile/energy/declone/declone_spider))
+		if (!degenerate)
+			if (spider_tier < 2)
+				if (ckey)
+					degenerate = 1
+				else
+					gib()
+			else
+				visible_message("<span class='danger'> \icon[src] [src] resists the bioweapon! </span>")
+	else if (istype(Proj, /obj/item/projectile/energy/declone))
+		if (!degenerate && prob(20))
+			visible_message("<span class='danger'> \icon[src] [src] looks staggered by the bioweapon! </span>")
+			if (spider_tier < 3)
+				degenerate = 1
+
+	..()
+
+
+// --------------------------------------------------------------------------------
+// --------------------- TERROR SPIDERS: PROC OVERRIDES ---------------------------
+// --------------------------------------------------------------------------------
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/examine(mob/user)
+	..()
+	var/msg = ""
+	if (src.stat == DEAD)
+		msg += "<span class='deadsay'>It appears to be dead.</span>\n"
+	else
+		if (src.key)
+			msg += "<BR><span class='warning'>Its eyes regard you with a curious intelligence.</span>"
+		if (src.ai_type == 0)
+			msg += "<BR><span class='warning'>It appears aggressive.</span>"
+		else if (src.ai_type == 0)
+			msg += "<BR><span class='notice'>It appears defensive.</span>"
+		else if (src.ai_type == 2)
+			msg += "<BR><span class='notice'>It appears passive.</span>"
+
+		if (health > (maxHealth*0.95))
+			msg += "<BR><span class='notice'>It is in excellent health.</span>"
+		else if (health > (maxHealth*0.75))
+			msg += "<BR><span class='notice'>It has a few injuries.</span>"
+		else if (health > (maxHealth*0.55))
+			msg += "<BR><span class='warning'>It has many injuries.</span>"
+		else if (health > (maxHealth*0.25))
+			msg += "<BR><span class='warning'>It is barely clinging on to life!</span>"
+		if (degenerate)
+			msg += "<BR><span class='warning'>It appears to be dying.</span>"
+		else if (health < maxHealth && regen_points > regen_points_per_kill)
+			msg += "<BR><span class='notice'>It appears to be regenerating quickly</span>"
+		if (killcount == 1)
+			msg += "<BR><span class='warning'>It is soaked in the blood of its prey.</span>"
+		else if (killcount > 1)
+			msg += "<BR><span class='warning'>It is soaked with the blood of " + num2text(killcount) + " prey it has killed.</span>"
+	to_chat(usr,msg)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/New()
+	..()
+	if (type == /mob/living/simple_animal/hostile/poison/terror_spider)
```

Comment:
don't write it at all
admins can figure out that they have done something horribly wrong on their own
just give the base type a very shiny "ERROR" icon with the name "debug path" or something